ui/src/components/NavLink.jsx

```diff
@@ -26,7 +26,8 @@ export default React.forwardRef((props, ref) => {
       </Link>
     );
   } else {
-    const href = absolutePath ? url.toString() : cleanDuplicateSlash(getBasename() + url.toString());
+    // Note: + '/' + is required here
+    const href = absolutePath ? url.toString() : cleanDuplicateSlash(getBasename() + '/' + url.toString());
```

Comment:
You add 1 more slash then replace it. Does it make sense?

Reply:
`getBaseName()` can return either 

`/abc/def`, or `/abc/def/`

We would have to cover both scenarios

Reply:
Those two options are possible because we leave the work of filling the `homepage` field, to the user.

The user can enter the homepage url with a trailing slash, or not... It's up to them, and we should make sure things work both way